views/resetPassword.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<!--[if lt IE 9]>
+	<script src="dist/html5shiv.js"></script>
+	<![endif]-->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no" />	
+	<title>Meaningful - Forgot Password</title>
+	<link href='http://fonts.googleapis.com/css?family=Lato:100,300,400,700|Merriweather:400,300italic,300,700,900' rel='stylesheet' type='text/css'>
+	<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+
+	<link rel="stylesheet" type="text/css" href="/stylesheets/main.css">
+	<link rel="stylesheet" type="text/css" href="/stylesheets/login.css">
+</head>
+<body>
+	<div class="wrapper">
+		<div class="centered">
+			<h2>Reset My Password</h2>
```

Comment:
I think the capitalization comment from above can apply here too. We could leave this one if we wanted to since it's not written as much like a sentence as the other h2 header, but I'd also be okay with changing this to `Reset my password` to stay consistent with the other header depending on how you guys feel.